### ISSUE TITLE
openssl manual page titles

### ISSUE THREAD

**Issue by petrovr** [openssl/openssl#14548]
Sample from a current page :
> .TH X25519 7 "2021-02-16" "1.1.1j" "OpenSSL"

According documentation format is 
> .TH title section date source manual 

I would like to request argument 5 to be merged with argument 4.As result argument 5 will be empty, i.e system will dispay "standard title of manual section". 

As alternative argument 5 could contain project but with suffix suitable for section.

For instance request inspired from https://securebox.termoneplus.com/man1/crl.1.html vs https://securebox.termoneplus.com/man1/ssh.1.html .

**Comment by richsalz** [comment]
I do not think this is possible without changing pod2man, which I am sure the project does not want to do.
I tried
```
 pod2man --name=$name --section=$section\$(MANSUFFIX) \\
                --center=OpenSSL --release="OpenSSL \$(VERSION)" $pod >\$\@
```
And ended up with:
```
; grep TH doc/man/man1/openssl.1                           
.\" titles (.TH), headers (.SH), subsections (.SS), items (.Ip), and index
.TH OPENSSL 1ossl "2021-04-21" "perl v5.26.1" "OpenSSL 3.0.0-alpha16-dev"
```

@petrovr, if you can post *exactly* what you want the TH line to look like, I will try to do it.  It might not be possible.

**Comment by nhorman** [comment]
Marking as inactive, to be closed at the end of 3.4 dev barring further input

**Comment by t8m** [comment]
Closing as the issue was labeled as inactive for extended period of time and there was no additional input.
Please add a comment if you have any further information to support reopening the issue.
